packages/react-vapor/src/components/modalWizard/ModalWizard.tsx

```diff
@@ -13,6 +14,7 @@ export interface ModalWizardProps
     extends Omit<IModalCompositeOwnProps, 'modalBodyChildren' | 'validateShouldNavigate'> {
     id: string;
     onFinish?: () => unknown;
```

Comment:
Since `onFinish` is returning `unknown` anyway, could we just allow using a promise for this one?

Reply:
I think this gives the component more flexibility if the user wanted to create a toast after its been completed for an alert and have the modal close instantly.

Another reason for not enforcing the `onFinish` to be a promise is that this would be considered a breaking change as well.

Reply:
You don't have to enforce a promise :thinking: 
you can execute `onFinish`, get the return value

Reply:
You don't have to enforce a promise :thinking: 

you can execute `onFinish`, get the return value